.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "docs/notebooks"]
 	path = docs/notebooks
 	url = https://github.com/StingraySoftware/notebooks.git
+	branch = new_fourier
```

Comment:
This is necessary because the notebooks @matteobachetti  added are currently only available on the `new_fourier` branch of that repository. Before merging this PR to `main`, https://github.com/StingraySoftware/notebooks/pull/57 should be merged, and then this `.gitmodules` should be updated to refer to the `main` branch of [notebooks](https://github.com/StingraySoftware/notebooks).

...yes, this is very confusing. I don't know of a better way to do it, though.

Reply:
I set a reminder to change the branch back to main before merging